WKData/Sources/WKData/Data Controllers/WKGrowthTasks/WKGrowthTasksDataController.swift

```diff
@@ -104,7 +104,7 @@ import Foundation
     }
 
     fileprivate func getMetadataObject(from image: WKImageRecommendationAPIResponse.ImageMetadata) -> WKImageRecommendation.ImageMetadata {
-        let metadata = WKImageRecommendation.ImageMetadata(descriptionUrl: image.descriptionUrl, thumbUrl: image.thumbUrl, fullUrl: image.fullUrl, originalWidth: image.originalWidth, originalHeight: image.originalHeight, mediaType: image.mediaType, description: image.description, author: image.author, license: image.license, date: image.date, caption: image.caption, categories: image.categories, reason: image.reason, contentLanguageName: image.contentLanguageName)
+        let metadata = WKImageRecommendation.ImageMetadata(descriptionUrl: image.descriptionUrl, thumbUrl: image.thumbUrl, fullUrl: image.fullUrl, originalWidth: image.originalWidth, originalHeight: image.originalHeight, mediaType: image.mediaType, description: image.description, author: image.author, license: image.license, date: image.date, caption: image.caption, categories: image.categories, reason: image.reason, contentLanguageName: image.contentLanguageName, sectionNumber: image.sectionNumber)
```

Comment:
I've been looking at this data across a couple of languages, and it seems like the `sectionNumber` is always null. The call I took from Android already returns the first section's wikitext, under `revisions.slots.main.content`. I think we should just past that through to avoid the extra call and pass in section 0 when needed. Let me know what you think!